config/prod.exs

```diff
@@ -10,6 +10,11 @@ import Config
 # which you should run after static files are built and
 # before starting your production server.
 config :sacastats, SacaStatsWeb.Endpoint, url: [host: "sacastats.com", port: 80]
+# watchers: [
```

Comment:
I don't think we want watchers in prod - everything will be packaged up into a release, so the file structure won't be the same either.

Reply:
Yeah I just copied and pasted this over from dev. I was trying to figure out how to just run the `build-prod.js` script (more preferably the npm buildProd script so it cleans the output first) when starting the Phoenix server up

Reply:
Is this something that can be done though?

Reply:
I think I'm a bit confused on why you'd want to run a build script when starting the server up. Do you mean when building a release? If so I'd look at the `aliases/0` fn in `mix.exs`, there are some examples there like `assets.deploy` that will run ESBuild and `phx.digest` (which might be the place you want to run the TS -> JS conversion)

Reply:
Ah, okay. I think that's what I was wanting.  So would I just change it to this:
```elixir
"assets.deploy": ["cmd --cd assets node build.js --deploy", "phx.digest", "cmd --cd assets npm run build"]
```
Which would end up running this script under `package.json`:
```JSON
"build": "rm -rf ../priv/static/assets/dist && node --experimental-json-modules build-prod.js"
```

Reply:
I haven't seen that `cmd` before, is it windows only? I might not be able to run it if that's the case

Reply:
Yeah, it seems strange that that's the term they use, but if you look at our `mix.exs` that's how they `cd`. I also wonder if the first one's `cd assets` means I would already be there and by the time the third command runs it won't be able to `cd` into `assets` as it's already there.